genesyscloud/routing_utilization_label/genesyscloud_routing_utilization_label_proxy.go

```diff
@@ -0,0 +1,142 @@
+)
+
+var internalProxy *routingUtilizationProxy
+
+type getAllRoutingUtilizationLabelsFunc func(ctx context.Context, p *routingUtilizationProxy, name string) (*[]platformclientv2.Utilizationlabel, *platformclientv2.APIResponse, error)
+type createRoutingUtilizationLabelFunc func(ctx context.Context, p *routingUtilizationProxy, req *platformclientv2.Createutilizationlabelrequest) (*platformclientv2.Utilizationlabel, *platformclientv2.APIResponse, error)
+type getRoutingUtilizationLabelFunc func(ctx context.Context, p *routingUtilizationProxy, id string) (*platformclientv2.Utilizationlabel, *platformclientv2.APIResponse, error)
+type getRoutingUtilizationLabelByNameFunc func(ctx context.Context, p *routingUtilizationProxy, name string) (*platformclientv2.Utilizationlabel, *platformclientv2.APIResponse, error)
+type updateRoutingUtilizationLabelFunc func(ctx context.Context, p *routingUtilizationProxy, id string, updateutilizationlabelrequest *platformclientv2.Updateutilizationlabelrequest) (*platformclientv2.Utilizationlabel, *platformclientv2.APIResponse, error)
+type deleteRoutingUtilizationLabelFunc func(ctx context.Context, p *routingUtilizationProxy, id string, forceDelete bool) (*platformclientv2.APIResponse, error)
+
+type routingUtilizationProxy struct {
+	clientConfig                         *platformclientv2.Configuration
+	routingApi                           *platformclientv2.RoutingApi
+	getAllRoutingUtilizationLabelsAttr   getAllRoutingUtilizationLabelsFunc
+	createRoutingUtilizationLabelAttr    createRoutingUtilizationLabelFunc
+	getRoutingUtilizationLabelAttr       getRoutingUtilizationLabelFunc
+	getRoutingUtilizationLabelByNameAttr getRoutingUtilizationLabelByNameFunc
+	updateRoutingUtilizationLabelAttr    updateRoutingUtilizationLabelFunc
+	deleteRoutingUtilizationLabelAttr    deleteRoutingUtilizationLabelFunc
+	routingCache                         rc.CacheInterface[platformclientv2.Utilizationlabel]
+}
+
+func newRoutingUtilizationProxy(clientConfig *platformclientv2.Configuration) *routingUtilizationProxy {
+	api := platformclientv2.NewRoutingApiWithConfig(clientConfig)
+	routingCache := rc.NewResourceCache[platformclientv2.Utilizationlabel]()
+	return &routingUtilizationProxy{
+		clientConfig:                         clientConfig,
+		routingApi:                           api,
+		getAllRoutingUtilizationLabelsAttr:   getAllRoutingUtilizationLabelsFn,
+		createRoutingUtilizationLabelAttr:    createRoutingUtilizationLabelFn,
+		getRoutingUtilizationLabelAttr:       getRoutingUtilizationLabelFn,
+		getRoutingUtilizationLabelByNameAttr: getRoutingUtilizationLabelByNameFn,
+		updateRoutingUtilizationLabelAttr:    updateRoutingUtilizationLabelFn,
+		deleteRoutingUtilizationLabelAttr:    deleteRoutingUtilizationLabelFn,
+		routingCache:                         routingCache,
+	}
+}
+
+func getRoutingUtilizationProxy(clientConfig *platformclientv2.Configuration) *routingUtilizationProxy {
```

Comment:
`getRoutingUtilizationProxy` -> `getRoutingUtilizationLabelProxy`

Reply:
This proxy has the same naming convention as in the routing_utilization package. Better off using `routingUtilizationLabel`.